### ISSUE TITLE
Applicant - "Choose one of the following..." or similar as default value for dropdowns

### ISSUE THREAD

**Issue by gggrant** [GCTC-NTGC/TalentCloud#603]
Resolved by #804 

# Description
Dropdown menus could easily be skipped for users with assistive technologies by having an accepted value as the default.

Suggest adding "Choose one of the following..." or similar as a default value for dropdown menus.

**Comment by gobyrne** [comment]
Notes from Chris: 
So our "quick fix" of adding `required` to the required select elements prevents the other forms on the page from submitting correctly because they're all technically the same form
Our "long fix" of adding appropriate form validation loses the user's input when validation fails
Our "extra long fix" of converting the whole thing to ajax submissions hasn't been done yet

**Comment by gggrant** [comment]
Many `<select>` elements don't have validation, or adding front-end validation breaks back-end functionality. So this turned out to be a larger task than anticipated.

As far as I'm aware:
- skills, references and work samples are good
- first page of application has validation

- degrees, courses, experience need work
- probably some parts of the manager side as well

**Comment by Khristophor** [comment]
Fixed up issues with Composer packages, I believe this is good to go
https://github.com/GCTC-NTGC/TalentCloud/pull/658

**Comment by gggrant** [comment]
Resolved by #804
